lib/datasource/index.ts

```diff
@@ -100,6 +101,7 @@ async function fetchReleases(
   config: PkgReleaseConfig
 ): Promise<ReleaseResult | null> {
   const { datasource } = config;
+  // istanbul ignore if
```

Comment:
These ignored statements should all be tested before merge

Reply:
The extra tests should handle all that, I believe.